locust/stats.py

```diff
@@ -639,11 +659,13 @@ def print_stats(stats):
     console_logger.info((" %-" + str(STATS_NAME_WIDTH) + "s %7s %12s %7s %7s %7s  | %7s %7s") % ('Name', '# reqs', '# fails', 'Avg', 'Min', 'Max', 'Median', 'req/s'))
     console_logger.info("-" * (80 + STATS_NAME_WIDTH))
     total_rps = 0
+    total_fps = 0
     total_reqs = 0
     total_failures = 0
     for key in sorted(six.iterkeys(stats)):
         r = stats[key]
         total_rps += r.current_rps
+        total_fps += r.current_fps
```

Comment:
This variable doesn't seem to be used, so it should be removed.

Reply:
Forgot to include it in the print_stats table. I'll do that instead.